test/utils.py

```diff
@@ -573,3 +573,14 @@ def wait_for_lines_in_output(stdout, expected_lines: list):
                 break
 
     return output
+
+
+@retry(Exception, tries=40, delay=0.5)
+def wait_pid_disappear(file, target_pid):
+    found = True
+    while found:
+        with open(file, "r") as f:
```

Comment:
Question: when the instance stopped a file `tarantool.pid` keeps on filesystem?
It seems, that if no a such file, retries for this function will takes 20 seconds.